src/libmongoc/tests/test-mongoc-write-concern.c

```diff
@@ -726,6 +721,24 @@ test_write_concern_inheritance_fam_txn (void *context)
    mongoc_find_and_modify_opts_destroy (opts);
 }
 
+static void
+test_fam_no_session_no_txn ()
```

Comment:
Use `(void)` to denote no arguments. https://stackoverflow.com/a/51080/774658

Reply:
Ah, that's interesting.  I didn't realize that.